server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckInController.java

```diff
@@ -38,6 +43,12 @@
     @Inject
     CheckInServices checkInservices ;
 
+    @Inject
+    CurrentUserServicesImpl currentUserServices;
```

Comment:
Not specific to this story, but a better practice is to make these fields private and inject them via the constructor.

Reply:
Also...why are you injecting the implementation rather than the interface here?

Reply:
good catch. I'll update it.